build/ci/nightly/Jenkinsfile

```diff
@@ -0,0 +1,58 @@
+pipeline {
+
+    agent {
+        label 'linux'
+    }
+
+    options {
+        timeout(time: 1, unit: 'HOURS')
+    }
+
+    environment {
+        VAULT_ADDR = credentials('vault-addr')
+        VAULT_ROLE_ID = credentials('vault-role-id')
+        VAULT_SECRET_ID = credentials('vault-secret-id')
+        GCLOUD_PROJECT = credentials('k8s-operators-gcloud-project')
+        REGISTRY = "push.docker.elastic.co"
+        REPOSITORY = "eck-snapshots"
+        IMG_NAME = "eck-operator"
+        IMG_SUFFIX = ""
+        SNAPSHOT_RELEASE = "true"
+        DOCKER_IMAGE_NO_TAG = "docker.elastic.co/${REPOSITORY}/${IMG_NAME}"
+    }
+
+    stages {
+        stage('Run unit and integration tests') {
+            steps {
+                sh 'make -C build/ci ci-pr'
+            }
+        }
+        stage('Build and push Docker image') {
+            steps {
+                sh """
+                    export TAG_NAME=\$(date +%F)
+                    make -C build/ci ci-release
+                """
+            }
+        }
+    }
+
+    post {
+        success {
+            script {
+                def version = sh(returnStdout: true, script: 'cat $WORKSPACE/operators/VERSION')
+                def date = new Date()
+                def image = env.DOCKER_IMAGE_NO_TAG + ":" + version + "-" + date.format("yyyy-MM-dd")
```

Comment:
Would it make sense to add another tag with the git hash? If we accidentally or deliberately trigger multiple builds within a single day, it would be useful to identify the different builds. Some users might also prefer to pin to a specific image so that they know exactly which code is running on their system.

Reply:
Fixed